app/presenters/features.rb

```diff
@@ -1,40 +1,41 @@
 # frozen_string_literal: true
 
 class Features
-  # This is for toggling Wiki Ed-specific features that should not be active
-  # on any other dashboard instances.
-  def self.wiki_ed?
-    return true if ENV['dashboard_url'] == 'dashboard.wikiedu.org'
-    return true if ENV['dashboard_url'] == 'dashboard-testing.wikiedu.org'
-    false
-  end
+  # Add methods which you want to be true for the respective configurations
 
-  def self.default_course_type
-    ENV['default_course_type'] || 'ClassroomProgramCourse'
-  end
+  PE = %i[
+    disable_help? disable_onboarding? wiki_trainings?
+    enable_language_switcher? enable_account_requests?
+    disable_wiki_output? enable_article_finder? open_course_creation?
+  ].freeze
 
-  def self.default_course_string_prefix
-    default_course_type.constantize.new.string_prefix
-  end
+  WIKI_ED = %i[
+    enable_get_help_button?
+    wiki_ed?
+  ].freeze
 
-  def self.disable_onboarding?
-    ENV['disable_onboarding'] == 'true'
-  end
+  is_ed = ENV['wiki_education'] == 'true'
 
-  def self.disable_help?
-    ENV['disable_help'] == 'true'
+  PE.each do |method|
+    define_singleton_method method do
```

Comment:
I would rather not do this with metaprogramming for this. Repeating `ENV['wiki_education'] == 'true'` isn't a big deal, and will keep the code extremely transparent.

Reply:
I did that first but it felt weird to have the same code repeated many times. Also thought since we are whitelisting the methods, there wouldn't be anything to worry about. But your point seems fair. Have refactored like you asked.